Dockerfile

```diff
@@ -44,8 +44,6 @@ MAINTAINER IPFS Stewards <w3dt-stewards-ip@protocol.ai>
 ENV GOPATH                 /go
 ENV SRC_PATH               /go/src/github.com/ipfs/bifrost-gateway
 ENV BIFROST_GATEWAY_PATH   /data/bifrost-gateway
-ENV STRN_LOGGER_URL        https://twb3qukm2i654i3tnvx36char40aymqq.lambda-url.us-west-2.on.aws
-ENV STRN_ORCHESTRATOR_URL  https://orchestrator.strn.pl/nodes/nearby?count=1000&core=true
```

Comment:
:information_source:  This removes remaining hardcoded URLs.

After this PR is merged, all `STRN_*` options need to be explicitly passed to docker image via `-e` now. 

 I'll set it up on bifrost-gw-staging, but need @gmasgras to update rollout env as well.